Backend/stream_mp3url.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 # Usage: ./stream_mp3url.sh <WSPort> <URL>
 
-curl --output - -L $2| sox -tmp3 - -traw -e signed-integer -b 8 -c 1 -r 96000 - | ./aucmp | wscat -b -l $1
+curl --output - -L $2| sox -tmp3 - -traw -e signed-integer -b 8 -c 1 -r 96000 - | ./aucmp | wscat -b -k -l $1
```

Comment:
Why are you adding the `-k` option?

If you're streaming an actual mp3 stream there shouldn't be an EOF and if you're streaming a "normal" mp3 file via URL we should probably close the websocket to ensure that the client side can stop playing.

Reply:
I thought adding the option would make my websocket to stop dying, I was trying to use local server as url and it just kept dying all the time, -k tho didn't fix the dying situation